.github/workflows/runnable_cxx.yml

```diff
@@ -221,7 +221,7 @@ jobs:
       uses: actions/cache@v1
       with:
         path: ${{ github.workspace }}/clang+llvm-${{ matrix.cxx-version }}-${{ matrix.arch }}/
-        key: ${{ matrix.cxx-version }}-${{ matrix.arch }}-1
+        key: ${{ matrix.cxx-version }}-${{ matrix.arch }}-3
```

Comment:
Switch back to -1?

Reply:
Why? The cache has been propagated, it doesn't really matter.

Reply:
I don't know which version of the script this is using. Maybe make it a timestamp just in case? `-20220925`

Reply:
Changed to use a timestamp, and we can't expire them AFAIK